dymos/phase/phase.py

```diff
@@ -932,191 +932,6 @@ def set_parameter_options(self, name, val=_unspecified, units=_unspecified, opt=
         if include_timeseries is not _unspecified:
             self.parameter_options[name]['include_timeseries'] = include_timeseries
 
-    def add_design_parameter(self, name, val=_unspecified, units=_unspecified, opt=_unspecified,
```

Comment:
These removed methods have to stay for now, for deprecation purposes.  We can yank them out in 1.0.0.